docs/Developers/Newcomers_Guide.rst

```diff
@@ -23,28 +23,28 @@ following tasks, you will become a developer at coala:
   - merge a ``difficulty/low`` Pull Request
   - review at least a ``difficulty/low`` or higher Pull Request
 
-When you ran coala on a project, please fill our
-`usability survey <http://coala.io/usability>`_. Once you got your first Pull
-Request merged successfully, fill in our
-`survey form <http://coala.io/newform>`_. With that you can help us making your
+Once you've run coala on a project, please fill out our
+`usability survey <http://coala.io/usability>`_. And once you've got your first Pull
+Request merged successfully, fill out our
+`survey form <http://coala.io/newform>`_. By doing so, you can help us make your
 experience better!
 
-Once you have achieved all these, just ask for being promoted on the chat and
-provide links to your reviews and merged Pull Requests. Then, you will be able
-to name yourself a coala developer!
+Once you have achieved all these, just ask to be promoted on the chat
+(see step 1 below) and provide links to your reviews and merged Pull Requests.
+Then you'll be able to call yourself a coala developer!
 
 .. note::
 
-    **Do not only fix a newcomer issue!** Supervising newcomers is really a lot
+    **Don't just fix a newcomer issue!** Supervising newcomers is really a lot
     of work. We're all volunteers and we can't keep this up if you don't help
     us in other areas as well!
 
-Of course, the order is not important, although, we recommend you to start
-with a ``newcomer`` issue, end with a ``low`` issue, and review other PRs in
-the meantime!
+Of course, the order of the steps above is not important, although we
+recommend you start with a ``newcomer`` issue, end with a ``low`` issue, and
```

Comment:
And please make sure you run `coala` before pushing commits. It will help you push your commits without any gitmate-bot errors and reduce cluttering over here.